src/Symfony/Component/Form/Forms.php

```diff
@@ -24,11 +24,12 @@
  * $formFactory = Forms::createFormFactory();
  *
  * $form = $formFactory->createBuilder()
- *     ->add('firstName', 'text')
- *     ->add('lastName', 'text')
- *     ->add('age', 'integer')
- *     ->add('gender', 'choice', array(
- *         'choices' => array('m' => 'Male', 'f' => 'Female'),
+ *     ->add('firstName', 'Symfony\Component\Form\Extension\Core\Type\TextType')
```

Comment:
I think we should use `::class` constants here. Almost nobody uses PHP < 5.5 nowadays anyway.